core/src/main/java/com/linecorp/armeria/server/cors/CorsServiceBuilder.java

```diff
@@ -16,28 +16,19 @@
 
 package com.linecorp.armeria.server.cors;
 
+import static com.google.common.base.Preconditions.checkState;
 import static java.util.Objects.requireNonNull;
 
-import java.util.Arrays;
 import java.util.Collections;
-import java.util.HashMap;
 import java.util.HashSet;
-import java.util.LinkedHashSet;
-import java.util.Map;
 import java.util.Set;
 import java.util.function.Function;
 import java.util.function.Supplier;
 
-import com.google.common.base.Ascii;
-
-import com.linecorp.armeria.common.HttpHeaderNames;
 import com.linecorp.armeria.common.HttpMethod;
 import com.linecorp.armeria.common.HttpRequest;
 import com.linecorp.armeria.common.HttpResponse;
 import com.linecorp.armeria.server.Service;
-import com.linecorp.armeria.server.cors.CorsConfig.ConstantValueSupplier;
-
-import io.netty.util.AsciiString;
 
 /**
  * Builds a new {@link CorsService} or its decorator function.
```

Comment:
Could you add a simple example that configures multiple origins with different policies? (using chaining)